domain/schema/controller.go

```diff
@@ -103,7 +104,8 @@ ON change_log_namespace (namespace);
 INSERT INTO change_log_namespace VALUES
     (1, 'external_controller'),
     (2, 'controller_node'),
-    (3, 'controller_config');
+    (3, 'controller_config'),
+	(4, 'model_migration');
```

Comment:
There needs to be 2:
- `model_migration_status` (watched to determine when migrations are active or not).
- `model_migration_minion_sync` (watched during migration to determine when all agents report success).